### ISSUE TITLE
The test ms-ssim metric seems lower than that in the paper

### ISSUE THREAD

**Issue by fenyying** [xiaosu-zhu/McQuic#1]
**Describe the bug**
Thanks for this great work. I am trying to test the compression performance using the pretrained model(qp_2_msssim_8e954998.mcquic). But the test ms-ssim metric seems lower than that in the paper. 

**To Reproduce**
`import os
os.environ['CUDA_VISIBLE_DEVICES'] = "0"
from cli import *
import torch
import torch.nn.functional as F
from torchvision.transforms.functional import convert_image_dtype
from mcquic.validate.utils import Decibel
import math
from demo import main
from mcquic.validate.metrics import MsSSIM
from torchvision.io.image import read_image, ImageReadMode, write_png

def psnr(a: torch.Tensor, b: torch.Tensor) -> float:
    mse = F.mse_loss(a, b).item()
    return -10 * math.log10(mse)

if __name__ == '__main__':
    ms_ssim = MsSSIM(4, data_range=1.0)
    with torch.inference_mode():

        input_name = './test_dataset/kodak/kodim{:02d}.png'.format(24)
        encode_name = './test_dataset/encode/kodim{:02d}.mcq'.format(24)
        decode_name = './test_dataset/decode/kodim{:02d}.png'.format(24)
        main(debug=False, quiet=False, qp=2, local=None, disable_gpu=False, mse=False, crop=True, input=pathlib.Path(input_name), output=pathlib.Path(encode_name))
        main(debug=False, quiet=False, qp=2, local=None, disable_gpu=False, mse=False, crop=True, input=pathlib.Path(encode_name), output=pathlib.Path(decode_name))
        device = torch.device("cpu")
        image_src = convert_image_dtype(read_image(input_name, ImageReadMode.RGB).to(device)) 
        image_dst = convert_image_dtype(read_image(decode_name, ImageReadMode.RGB).to(device))
        image_src = image_src.unsqueeze(0)
        image_dst = image_dst.unsqueeze(0)
        cur_psnr = psnr(image_src, image_dst)
        cur_ms_ssim = Decibel(1.0)(ms_ssim(image_src, image_dst)).item()
        print('psnr: ', cur_psnr, ', ms_ssim: ', cur_ms_ssim)`

**Expected behavior**
**The program output:**
                                                                                                                                                                         
psnr:  23.59759484753215 , ms_ssim:  12.082359313964844

The result in the paper:
![image](https://user-images.githubusercontent.com/42135636/161049675-17f9d632-c509-44c8-b6a7-2d093eae4df1.png)

12.0823 < 13.52
Could you explain it？

**Comment by xiaosu-zhu** [comment]
Hi @fenyying ,

Thanks for your report and attention to our work. You may notice that this code repository has been totally refactored since commit [cac2921b7c](https://github.com/xiaosu-zhu/McQuic/tree/cac2921b7c). The model structure, training loop, etc. are all refreshed to make the code clean. Therefore current model somehow has different performance than before. I'm still investigating it.

To reproduce result in paper, I provide the original model here ([encoder.pt](https://github.com/xiaosu-zhu/McQuic/releases/download/generic/encoder.pt), [decoder.pt](https://github.com/xiaosu-zhu/McQuic/releases/download/generic/decoder.pt)). You could run this model with legacy code by checking out commit `cac2921b7c`. And run `python src/mcqc/evaluation/eval.py --cfg configs/ssim/v100/12-13+11+9.yaml --dataset data/kodak/ --encoder encoder.pt --decoder decoder.pt`.

The results will produces in a folder named `ckpt/images` like this:

![image](https://user-images.githubusercontent.com/11695585/161186062-493c210e-a9f6-41da-be6e-abed213264c1.png)

where the file with suffix `23` is exactly like this:
![image](https://user-images.githubusercontent.com/11695585/161186156-33d7064d-786e-419f-a49d-32127c77c6be.png)
which has `MS-SSIM=13.51dB`.

But I still recommend you to wait for me investigating why current model has slightly lower performance on Kodak (As for on CLIC, performance is good as before). I think it may be caused by some data augmentation strategy fails. I will release updated model as soon as I solve this bug. Thanks again.

**Comment by fenyying** [comment]
Thanks a lot.
